webauthn-rs/src/lib.rs

```diff
@@ -562,10 +563,10 @@ impl Webauthn {
     ///
     /// This function returns a `CreationChallengeResponse` which you must serialise to json and
     /// send to the user agent (e.g. a browser) for it to conduct the registration. You must persist
-    /// on the server the `SecurityKeyRegistration` which contains the state of this registration
+    /// on the server the [SecurityKeyRegistration] which contains the state of this registration
     /// attempt and is paired to the `CreationChallengeResponse`.
     ///
-    /// WARNING ⚠️  YOU MUST STORE THE `SecurityKeyRegistration` VALUE SERVER SIDE.
+    /// WARNING ⚠️  YOU MUST STORE THE [SecurityKeyRegistration] VALUE SERVER SIDE.
```

Comment:
There is more than just the SecurityKeyRegistration type, isn't there also Passkey and PasswordlessKey?

Reply:
Yep, but I'm pretty clueless on that front, just did some quick changes to get the work started and show how it's done. 😄

Reply:
I'll do the rest then :)